examples/elk/Vagrantfile

```diff
@@ -1,7 +1,7 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-converge_version = "0.2.0"
+converge_version = "0.3.0"
```

Comment:
I figured it was better to set that now since the examples are built to download the release binaries and will no longer work with the 0.2.0 binary. All in all, I think we need a better system for the examples. I will open an issue on that.